Package.swift

```diff
@@ -45,9 +45,8 @@ let package = Package(
       dependencies: [
         .product(name: "GRPC", package: "grpc-swift"),
         .product(name: "FirebaseCore", package: "firebase-ios-sdk"),
-        // TODO: Investigate switching Auth and AppCheck to interop.
+        // TODO: Investigate switching Auth to interop.
         .product(name: "FirebaseAuth", package: "firebase-ios-sdk"),
-        .product(name: "FirebaseAppCheck", package: "firebase-ios-sdk"),
         .product(name: "GULEnvironment", package: "GoogleUtilities"),
```

Comment:
Add a dependency for FirebaseAppCheckInterop so it's not relying upon its transitive dependency through Auth

Reply:
Maybe we could use it transitively via Auth for now and then switch to it when we decide on the course forward.

Reply:
SGTM

Reply:
Would you add a comment/TODO?

Reply:
Added a TODO in Package.swift.